examples/elemental/build/release.yaml

```diff
@@ -1,2 +1,4 @@
 name: suse-edge
 manifestURI: oci://registry.opensuse.org/home/adinov/core/containerfile/release-manifest:0.0.1
+enable:
+  - neuvector
```

Comment:
I can easily see this growing to have different types of enablements, for example:
```
enable:
   helm:
       - neuvector
    system:
       - dpdk
       - ...
```

However, it probably makes more sense to adapt to that closer to our "custom extensions" story rather than now.

Reply:
In fact, this is very likely changing in an upcoming PR to allow for custom values.